src/services/user/services/RegistrationLink.js

```diff
@@ -0,0 +1,91 @@
+const { BadRequest, Forbidden } = require('@feathersjs/errors');
+
+const { SC_SHORT_TITLE, SC_DOMAIN } = require('../../../../config/globals');
+
+const mailContent = (firstName, lastName, registrationLink) => {
+	const mail = {
+		subject: `Einladung für die Nutzung der ${SC_SHORT_TITLE}!`,
+		content: {
+			text: `Einladung in die ${SC_SHORT_TITLE}
+    Hallo ${firstName} ${lastName}!
+    \\nDu wurdest eingeladen, der ${SC_SHORT_TITLE} beizutreten,
+    \\nbitte vervollständige deine Registrierung unter folgendem Link: ${registrationLink}
+    \\nViel Spaß und einen guten Start wünscht dir dein ${SC_SHORT_TITLE}-Team`,
+			html: '',
+		},
+	};
+	return mail;
+};
+
+const { userModel } = require('../model');
+
+const getCurrentUserInfo = (id) => userModel.findById(id)
+	.select(['schoolId', 'email', 'firstName', 'lastName'])
+	.populate('roles')
+	.lean()
+	.exec();
+
+class RegistrationLink {
+	async create(data, params) {
+		let totalMailsSend = 0;
+
+		try {
+			const { userIds } = data;
+
+			for (const userId of userIds) {
+				const account = await this.app.service('/accounts').find({
+					query: {
+						userId,
+					},
+				});
+
+				// dont't send mail if user has account
+				if ((account || []).length === 0) {
+					// get user info from id in userIds
+					const user = await Promise.resolve(getCurrentUserInfo(userId));
```

Comment:
why Provise.resolve? If it is a Promise await will be enough and errors with catched by the catch block